### ISSUE TITLE
Does is still work?

### ISSUE THREAD

**Issue by Kevsosmooth** [natecox/pyshipstation#40]
Is this still supported because I have not used shipstation api before and need a solution for this to create some labels faster than normal ? I know shipstation has had a bunch of updates... so just checking

**Comment by agritheory** [comment]
@Kevsosmooth So I'm listed as a maintainer here and have been meaning to do a write up on a V2 of this package and potential integrations with ShipEngine (ShipStation's new backend, though they're not really saying it that way). I had a meeting with some folks over there a few weeks ago and I think it's time to organize it all. My hope is to pull the features from [this version](https://github.com/agritheory/shipstation-client) in and get it to be compatible with the new APIs.

**Comment by Kevsosmooth** [comment]
makes sense... so this means that this code is no longer valid correct because I did download the package just did not want to start if it was not going to work on my end. I know shipstation has python support this just seemed so much smoother.

**Comment by Kevsosmooth** [comment]
I do notice you can download the v1 api key so im assuming it would work with that one correct?

**Comment by agritheory** [comment]
I don't think the V2 API will in general availability until sometime Q1 2025 or later and I expect the V1 API to remain stable but not with feature parity to V2 for quite a while. V2 has some nice new features - mostly better many:many models between item, order and shipment.

**Comment by Kevsosmooth** [comment]
I see so I used your code would you be able to let me know what I did wrong on this coding part. I ran the code and no errors but i do not see it on shisptatiion just wondering if its because it on the v2 engine.

`from shipstation.api import *

api_key = "123"
api_secret = "ABC"

ss = ShipStation(key=api_key, secret=api_secret)

id = '[hacked123]'
ss_order = ShipStationOrder(order_number=id)
ss_order.set_status('awaiting_shipment')
ss_order.set_customer_details(
    username='foobar',
    email='foo@bar.org'
)
shipping_address = ShipStationAddress(
    name='[TEST 123]',
    street1='[street line 2]',
    city='[QUEENS]',
    state='[New York]',
    postal_code='[11111]',
    country = '[US]'


)


ss_order.set_shipping_address(shipping_address)



ss.add_order(order=ss_order)
ss.submit_orders()`

**Comment by Kevsosmooth** [comment]
Anyone you would have a working example so i can reference it because i tried a few different ways and to be honest nothing.

**Comment by Kevsosmooth** [comment]
Ok got it to work but now my issue is I typed in both gb and or uk and nothing

**Comment by Kevsosmooth** [comment]
self.country = country


You guys did not add this piece of code to to it so i had to add it to be able to put international address.

**Comment by strefli3** [comment]
@agritheory any updates on your V2 progress?
